platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -174,6 +174,8 @@ public interface CoreI18n extends EmfI18n {
 		.de("Kern-Sprache");
 	I18n0 CORE_LANGUAGES = new I18n0("Core Languages")//
 		.de("Kern-Sprachen");
+	I18n0 CORE_MODULE_ADMINISTRATORS_CANNOT_BE_IMPERSONATED = new I18n0("Core module administrators cannot be impersonated.")//
```

Comment:
This terminology quires discussion

Reply:
Btw, this functionality has been inaccessible for like 2yrs now. We should just delete it.